### PR TITLE
Update asyncio loop set default executor

### PR DESCRIPTION
**Pull request by sengineer0**
…tor to asyncio.loop.set_default_executor

Ref: https://github.com/biothings/biothings.api/issues/153

### CODE REVIEW

File: tests/utils/test_manager.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        # Action
+        manager = JobManager(loop)
+
+        # Asserts
+        assert loop == manager.loop
+        assert isinstance(manager.loop._default_executor, ThreadPoolExecutor)
+
+        # Given
+        thread_executor = ThreadPoolExecutor()
+        process_executor = ProcessPoolExecutor
```
Comment (newgene) [inline tests/utils/test_manager.py]: missing `()` at the end?

File: biothings/utils/manager.py [hunk 2]
```diff
@@ -466,15 +466,7 @@ def __init__(self, loop, process_queue=None, thread_queue=None, max_memory_usage
...
-        # FIXME: In Py38 using an executor that is not a ThreadPoolExecutor is
-        #  deprecated. And it seems in Py39 , it must be a ThreadPoolExecutor,
-        #  using a ProcessPoolExecutor will trigger an error.
-        #  However, loop.run_in_executor still accepts ProcessPoolExecutor
-        #  see https://bugs.python.org/issue34075
-        if default_executor == "thread":
-            self.loop.set_default_executor(self.thread_queue)
-        else:
-            self.loop.set_default_executor(self.process_queue)
+        self.loop.set_default_executor(self.thread_queue)
```
Comment (newgene) [inline biothings/utils/manager.py:474]: include some comments to explain why only `thread_queue` can be used here.

File: biothings/hub/__init__.py [hunk 3]
```diff
@@ -104,7 +104,7 @@ def _config_for_app(config_mod=None):
 
 def get_loop(max_workers=None):
     loop = asyncio.get_event_loop()
-    executor = concurrent.futures.ProcessPoolExecutor(max_workers=max_workers)
+    executor = concurrent.futures.ThreadPoolExecutor(max_workers=max_workers)
```
Comment (newgene) [inline biothings/hub/__init__.py:107]: With this change to `ThreadPoolExecutor`, did you test it works as expected?

Comment (sengineer0) [reply biothings/hub/__init__.py:107]: Yes it works. atm no where call this method
So I did a test by doing this method: in `biothings/hub/__init__.py::configure_job_manager`, when create a `JobManager` instance, I passed in `get_loop(max_workers)`, then retest the flow, and I saw the code still work.

**Pull request merged**